src/components/widgets/FieldInput.jsx

```diff
@@ -30,14 +30,16 @@ const FieldInput = ({
 
   const validations = (
     <div className="div-error">
-      {(
+      {error && (
         <span className="field-error" style={{ visibility: error ? "visible" : "hidden"}}>
           { error ? error : "_" }
-        </span>) ||
-      (
```

Comment:
@rhenshaw56 I believe that @cioan deliberately changed this because we wanted the span always present so that the form layout doesn't change when an error is rendered. (ie, field below the field with the error is "pushed down" when an error is rendered)

However the "visibility" should take care hiding/showing the span... this worked with the other app we are building, but maybe we are missing something? Can you explain what is happening in that case?  Thanks!

Reply:
@mogoodrich the `||` condition on line 36 that was making the warning class not show at all, so in that case only extreme errors are shown (which don't validate for abnormal results) and not warnings

Reply:
@rhenshaw56 , @cioan  is taking a look now...

Reply:
@cioan @rhenshaw56  to achieve the result we are looking for, maybe we have a single span with visibility set on warning or error?  And then change the className based on whether it's an error or a warning, with the error taking precedence?